Products/ZenModel/migrate/UpdateRolesForDefineCommands.py

```diff
@@ -0,0 +1,38 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2020, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+
+__doc__ = '''
+This migration script puts CZ_ADMIN_ROLE and ZEN_MANAGER_ROLE roles to the ZEN_DEFINE_COMMANDS_EDIT permission.
+'''
+
+
+import Migrate
+
+from Products.ZenModel.ZenossSecurity import (
+    CZ_ADMIN_ROLE,
+    MANAGER_ROLE,
+    ZEN_MANAGER_ROLE,
+    ZEN_DEFINE_COMMANDS_EDIT
+)
+from Products.ZenModel.migrate.UpdateDefineCommandsEditPermission import UpdateDefineCommandsEditPermission
+from Products.ZenModel.ZMigrateVersion import SCHEMA_MAJOR, SCHEMA_MINOR, SCHEMA_REVISION
+
+
+class UpdateRolesForDefineCommands(Migrate.Step, UpdateDefineCommandsEditPermission):
```

Comment:
The `Migrate.Step` class may be doing something unconventional that prevents more than one class inheriting from it in the same class hierarchy.

You could try fixing `Migrate.Step` (or the metaclass it specifies), but an alternative fix is to not inherit from `UpdateDefineCommandsEditPermission` class and invoke the `addPermission` method manually, e.g.
```
    UpdateDefineCommandsEditPermission.addPermission(
        self,
        dmd.zport,
        ZEN_DEFINE_COMMANDS_EDIT,
        [MANAGER_ROLE, CZ_ADMIN_ROLE, ZEN_MANAGER_ROLE],
        1,
    )
```

Reply:
fixed using alternative approach